### PR TITLE
docs(deploy): token config of hexo-deployer-git

### PR DESCRIPTION
**Pull request by curbengh**
- [x] Others (Update, fix, translation, etc...)

`token` option is introduced in https://github.com/hexojs/hexo-deployer-git/pull/135

https://github.com/hexojs/hexo-deployer-git/pull/151#issuecomment-552089551

### CODE REVIEW

**Comment by curbengh** [comment]
Pending publish of new version of hexo-deployer-git.

File: source/docs/one-command-deployment.md [hunk 1]
```diff
@@ -43,18 +47,17 @@ deploy:
...
+| `token`   | Optional token value to authenticate with the repo. Prefix with `$` to read token from environment variable |
 
-Your repository will have a **master** branch when you first made it. Keep working on this branch to create your site. When you deploy Hexo will create, or update, a new branch on the remote site (called **published** in the config above). Deployment won't create a new branch locally, nor will it mess with your existing source code in the master branch locally or on the remote. You still need to keep pushing commits to the master branch manually to the remote server to keep your site backed up. In addition, if you are using a CNAME file to customize your Github Pages domain name, you need to put the CNAME file under `source_dir` so that Hexo can push it to the published branch.
+3. Deploy your site, `./node_modules/.bin/hexo clean && ./node_modules/.bin/hexo deploy` (or `hexo clean && hexo deploy` if you installed Hexo globally).
+  - You will be prompted with username and password of the target repository, unless you authenticate with a token.
```
Comment (curbengh) [inline source/docs/one-command-deployment.md]: not familiar with hexo-deployer-git, is this sentence correct?

File: source/docs/one-command-deployment.md [hunk 2]
```diff
@@ -43,18 +47,17 @@ deploy:
...
+| `branch`  | Branch name. Defaults to `gh-pages` on GitHub and `master` for others |
+| `message` | Customize commit message. Defaults to `Site updated: {% raw %}{{ now('YYYY-MM-DD HH:mm:ss') }}{% endraw %}` |
+| `token`   | Optional token value to authenticate with the repo. Prefix with `$` to read token from environment variable |
 
-Your repository will have a **master** branch when you first made it. Keep working on this branch to create your site. When you deploy Hexo will create, or update, a new branch on the remote site (called **published** in the config above). Deployment won't create a new branch locally, nor will it mess with your existing source code in the master branch locally or on the remote. You still need to keep pushing commits to the master branch manually to the remote server to keep your site backed up. In addition, if you are using a CNAME file to customize your Github Pages domain name, you need to put the CNAME file under `source_dir` so that Hexo can push it to the published branch.
+3. Deploy your site, `./node_modules/.bin/hexo clean && ./node_modules/.bin/hexo deploy` (or `hexo clean && hexo deploy` if you installed Hexo globally).
```
Comment (SukkaW) [inline source/docs/one-command-deployment.md]: Maybe just `Deploy your site using hexo clean && hexo deploy` is ok?

File: source/docs/one-command-deployment.md (hunk 1 above)
Comment (SukkaW) [reply source/docs/one-command-deployment.md]: Usually we setup hexo-deployer-git with https git url and it requires prompt. I don't know what will happen if I set up with ssh git url.

File: source/docs/one-command-deployment.md [hunk 3]
```diff
@@ -43,18 +47,17 @@ deploy:
...
+  - You will be prompted with username and password of the target repository, unless you authenticate with a token.
+  - hexo-deployer-git does not store your username and password. Use [git-credential-cache](https://git-scm.com/docs/git-credential-cache) to store them temporarily.
+4. Navigate to your repository settings and change the "Pages" branch to `published` (or the branch specified in your config). The deployed site should be live on the link shown on the "Pages" setting.
```
Comment (SukkaW) [inline source/docs/one-command-deployment.md]: GitHub only support publish site to GitHub Pages from:

- root directory of gh-pages branch
- root directory of master branch
- docs directory of master branch

And by default, hexo-deployer-git use those branches if no `branch` option is given:

- GitHub.com: gh-pages branch
- Coding.net: coding-pages branch
- Others: master branch

File: source/docs/one-command-deployment.md (hunk 1 above)
Comment (curbengh) [reply source/docs/one-command-deployment.md]: Mentioned `ssh key'.

File: source/docs/one-command-deployment.md (hunk 3 above)
Comment (curbengh) [reply source/docs/one-command-deployment.md]: Add default branches

**Comment by curbengh** [comment]
https://github.com/hexojs/hexo-deployer-git/pull/135 has been released in [v2.1.0](https://github.com/hexojs/hexo-deployer-git/pull/152).

**Pull request merged**